configure.ac

```diff
@@ -483,6 +483,12 @@ AC_SUBST([PART_LDFLAGS])
 CFLAGS=$SAVE_CFLAGS
 LDFLAGS=$SAVE_LDFLAGS
 
+# libblockdev swap
+SWAP_CFLAGS="-I/usr/include/blockdev"
+SWAP_LDFLAGS="-lbd_swap"
+AC_SUBST([SWAP_CFLAGS])
+AC_SUBST([SWAP_LDFLAGS])
+
```

Comment:
If I'm correct the libblockdev libraries/headers may be installed independently. I know the rest of the configure script contains similar things all over but I would really prefer AC_CHECK_*() or PKG_CHECK_MODULES() so the compilation fails early enough when the dependecies are missing. Moreover there is also a SPEC file template in packaging/, its BuildRequires should be updated too.